em++.bat

```diff
@@ -1,11 +1,11 @@
-@echo off
 :: Entry point for running python scripts on windows systems.
 :: To modify this file, edit `tools/run_python.bat` and then run
 :: `tools/create_entry_points.py`
 
-set EM_PY=%EMSDK_PYTHON%
-if "%EM_PY%"=="" (
+@setlocal
+@set EM_PY=%EMSDK_PYTHON%
+@if "%EM_PY%"=="" (
   set EM_PY=python
```

Comment:
Should this line also start with `@`?

Reply:
It does not. Windows actually sees the code as
```
@if "%EM_PY%"=="" ( set EM_PY=python )
```
as one construct, so it does not echo the set from inside.